.dev/tests/a11y/pa11y.js

```diff
@@ -32,16 +32,21 @@ if ( process.argv[ 2 ] ) {
 // Set up the pa11y config options
 const config = {
 	standard: packageJson.testing.accessibility.compliance,
-	hideElements: '#wpadminbar',
+	hideElements: '#wpadminbar, #gist5332801 > div > div:nth-child(2)',
```

Comment:
This is the gist block navigation at the bottom. The embed does not pass a11y tests. Since the frontend markup is out of our control, I am just hiding it in the test.
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/5321364/199587525-0d42c69c-0749-4cbc-a369-1951b95071d6.png">